### PR TITLE
Ingesting SyntaxFest 2019 (4 volumes)

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/xml/W19.xml [hunk 1]
```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
...
+    </meta>
+    <frontmatter></frontmatter>
+    <paper id="1">
+      <title><fixed-case>S</fixed-case>yntax<fixed-case>F</fixed-case>est 2019 Invited talk - Inductive biases and language emergence in communicative agents</title>
+      <author><first>Emmanuel</first><last>Dupoux</last></author>
+      <pages>1–1</pages>
+      <url>W19-7701</url>
+    </paper>
+    <paper id="2">
+      <title><fixed-case>S</fixed-case>yntax<fixed-case>F</fixed-case>est 2019 Invited talk - Transferring <fixed-case>NLP</fixed-case> models across languages and domains</title>
```
Comment (aryamccarthy) [inline data/xml/W19.xml]: Should invited talks get anthology entries?

File: data/xml/W19.xml [hunk 2]
```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
...
+    </paper>
+    <paper id="13">
+      <title>How to Parse Low-Resource Languages: Cross-Lingual Parsing, Target Language Annotation, or Both?</title>
+      <author><first>Ailsa</first><last>Meechan-Maddon</last></author>
+      <author><first>Joakim</first><last>Nivre</last></author>
+      <pages>112–120</pages>
+      <url>W19-7713</url>
+    </paper>
+    <paper id="14">
+      <title>Word order variation in Mbyá Guaraní</title>
```
Comment (aryamccarthy) [inline data/xml/W19.xml]: The language name should be fixed-case.

File: data/xml/W19.xml [hunk 3]
```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
...
+    <paper id="23">
+      <title>Presenting <fixed-case>TWITTIRÒ</fixed-case>-<fixed-case>UD</fixed-case>: An <fixed-case>I</fixed-case>talian Twitter Treebank in Universal Dependencies</title>
+      <author><first>Alessandra Teresa</first><last>Cignarella</last></author>
+      <author><first>Cristina</first><last>Bosco</last></author>
+      <author><first>Paolo</first><last>Rosso</last></author>
+      <pages>190–197</pages>
+      <url>W19-7723</url>
+    </paper>
+    <paper id="24">
+      <title>Pān.inian Syntactico-Semantic Relation Labels</title>
```
Comment (aryamccarthy) [inline data/xml/W19.xml]: Should be Pāṇinian (note the n with the dot beneath).

File: data/xml/W19.xml [hunk 4]
```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
...
+    <paper id="2">
+      <title>Are formal restrictions on crossing dependencies epiphenominal?</title>
+      <author><first>Himanshu</first><last>Yadav</last></author>
+      <author><first>Samar</first><last>Husain</last></author>
+      <author><first>Richard</first><last>Futrell</last></author>
+      <pages>2–12</pages>
+      <url>W19-7802</url>
+    </paper>
+    <paper id="3">
+      <title>A Surface-Syntactic <fixed-case>UD</fixed-case> Treebank for Naija</title>
```
Comment (aryamccarthy) [inline data/xml/W19.xml]: again, capitalized language name

File: data/xml/W19.xml [hunk 5]
```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
...
+      <author><first>Bernard</first><last>Caron</last></author>
+      <author><first>Marine</first><last>Courtin</last></author>
+      <author><first>Kim</first><last>Gerdes</last></author>
+      <author><first>Sylvain</first><last>Kahane</last></author>
+      <pages>13–24</pages>
+      <url>W19-7803</url>
+    </paper>
+    <paper id="4">
+      <title>Can Greenbergian universals be induced from language networks?</title>
```
Comment (aryamccarthy) [inline data/xml/W19.xml]: Greenbergian should be fixed-case

File: data/xml/W19.xml [hunk 6]
```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
...
+      <title>Linked Open Treebanks. Interlinking Syntactically Annotated Corpora in the <fixed-case>XXX</fixed-case> Knowledge Base of Linguistic Resources for <fixed-case>L</fixed-case>atin</title>
+      <author><first>Francesco</first><last>Mambrini</last></author>
+      <author><first>Marco</first><last>Passarotti</last></author>
+      <pages>74–81</pages>
+      <url>W19-7808</url>
+    </paper>
+    <paper id="9">
+      <title>Challenges of Annotating a Code-Switching Treebank</title>
+      <author><first/><last>Özlem Çetinoğlu</last></author>
+      <author><first/><last>ÇağrıÇöltekin</last></author>
```
Comment (aryamccarthy) [inline data/xml/W19.xml]: This should be <first> Çağrı</first><last>Çöltekin</last>

Comment (aryamccarthy) [reply data/xml/W19.xml]: And above, it should be `<first> Özlem</first><last> Çetinoğlu</last>`

File: data/xml/W19.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/W19.xml]: Can you pull down this branch and add these missed entries to `bin/fixedcase/truelist`?

File: data/xml/W19.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/W19.xml]: I guess, if there's an associated PDF.

**Comment by mjpost** [comment]
Thanks for the detailed comments!

File: data/xml/W19.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/W19.xml]: Actually I did it

File: data/xml/W19.xml (hunk 3 above)
Comment (mjpost) [reply data/xml/W19.xml]: Good eye, thank you!

**Comment by mjpost** [comment]
Does this look good now, @aryamccarthy?

**Comment by mjpost** [comment]
Thank you!

**Pull request merged**